src/segmantic/commands/monai_unet_cli.py

```diff
@@ -1,28 +1,23 @@
-import json
 import inspect
-import typer
+import json
+from functools import partial
 from pathlib import Path
 from typing import List
 
-from ..util.encoders import PathEncoder
+import typer
+import yaml
+
 from ..prepro.labels import load_tissue_list
 from ..seg import monai_unet
+from ..util.cli import get_default_args, validate_args
 
 app = typer.Typer()
 
 
-def _is_path(param: inspect.Parameter) -> bool:
-    if param.annotation != inspect.Parameter.empty and inspect.isclass(
-        param.annotation
-    ):
-        return issubclass(param.annotation, Path)
-    return False
-
-
 def _get_nifti_files(dir: Path) -> List[Path]:
     if not dir:
```

Comment:
MINOR: if ``not dir`` can happen, then ``dir: Optional[Path]``? 
Because ``Path()`` is the current working directory and I do not think it evaluates as false

Reply:
Default is `None`:
```
    labels_dir: Path = typer.Option(
        None,
        "--labels-dir",
        "-l",
        help="directory containing labelfields",
    ),
```